pkg/fakerp/server.go

```diff
@@ -126,44 +118,61 @@ func (s *Server) validate(w http.ResponseWriter, r *http.Request) bool {
 	return true
 }
 
-func (s *Server) restore(w http.ResponseWriter, path string) bool {
-	if _, err := os.Stat(path); err != nil {
-		return true
+// The way we run the fake RP during development cannot really
+// be consistent with how the RP runs in production so we need
+// to restore the internal state of the cluster from the
+// filesystem.
+func (s *Server) restore(path string) error {
+	dataDir, err := FindDirectory(path)
+	if err != nil {
+		return err
 	}
-	data, err := ioutil.ReadFile(path)
+	data, err := ioutil.ReadFile(dataDir)
```

Comment:
you should read the containerservice.yaml and not path (which is most likely _data)

Reply:
good catch, fixed